scripts/MRK17_Exp1_rk01.jmd

```diff
@@ -17,24 +17,27 @@ using DrWatson
 
 using MixedModels
 using CSV, DataFrames, DataFramesMeta, RCall 
-using Distributions: mean
+using Statistics: mean
```

Comment:
`mean` is defined in the `Statistics` package and imported into other packages from there.  I should have added `std` there too.  We do have a dependency on `Distributions` but it would be good to avoid having it explicit here because it is a heavyweight package.